README.md

```diff
@@ -76,3 +76,23 @@ forge script --broadcast --rpc-url <RPC-URL> --private-key <PRIVATE_KEY> --verif
 ## Acknowledgments
 
 Inspired by [merklejerk](https://github.com/merklejerk)'s [permit-everywhere](https://github.com/merklejerk/permit-everywhere) contracts which introduce permit based approvals for all tokens regardless of EIP2612 support.
+
+## ZK-Sync Era setup test
```

Comment:
Please follow the text style from the original code, here should be new line.

Reply:
Sure, i will do